src/Services/MediaLibrary/Glide.php

```diff
@@ -273,4 +269,38 @@ protected function getFocalPointCrop($crop_params, $width, $height)
 
         return [];
     }
+
+    /**
+     * @param string $id 
+     * @return string
+     */
+    private function getOriginalMediaUrl($id)
+    {
+        $libraryDisk = $this->config->get('twill.media_library.disk');
+        $endpointType = $this->config->get('twill.media_library.endpoint_type');
+        $localMediaLibraryUrl = $this->config->get('filesystems.disks.twill_media_library.url');
```

Comment:
```suggestion
        $localMediaLibraryUrl = $this->config->get("filesystems.disks.$libraryDisk.url");
```

Thanks for addressing the feedback so quickly! This should allow a custom media library disk to work with this implementation.